diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -146,6 +147,16 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
+    private fun isReferenceExpressionWithSameName(node: ASTNode, kdocTags: Collection<KDocTag>?): Boolean {
+        if (node.isSingleLineGetterOrSetter()) {
```

Comment:
Why is this check here? I think we have it somewhere else too

Reply:
Yes, removed